forgit.plugin.zsh

```diff
@@ -128,7 +128,12 @@ forgit::stash::show() {
         +s +m -0 --tiebreak=index --bind=\"enter:execute($cmd | LESS='-R' less)\"
         $FORGIT_STASH_FZF_OPTS
     "
-    git stash list | FZF_DEFAULT_OPTS="$opts" fzf --preview="$cmd"
+    if [ "$#" -eq 0 ]; then
+      git stash list | FZF_DEFAULT_OPTS="$opts" fzf --preview="$cmd"
+    else
+      # Prepend git stash list output with stash@{0}:..stash@{N-1}: to make the `cut` above works.
+      git stash list $* | awk '{print "stash@{" NR-1 "}:" $0}' | FZF_DEFAULT_OPTS="$opts" fzf --preview="$cmd"
+    fi
```

Comment:
Thanks for your work and the good idea to improve `gss` @attomos.
 
Support all options may easily break the output format pattern we depends to show preview.

Since other options may not very useful for `gss` we can improve the output format by pre-set the `--format` option.
```suggestion
    git stash list --color=always --format='%C(cyan)%gd%C(reset): %s %C(black bold)%cr%C(reset)' |
        FZF_DEFAULT_OPTS="$opts" fzf --preview="$cmd"
```

In this way we don't need to manually prepend stash ids. It shoud be more robust. Relative to flexibility, I prefer stability. How do you think about it? @attomos

Reply:
Or can we have a better way to get the stash id instead of manually calculating which I think is not very stable? So that can safely support any options for `stash list`.

Reply:
Thanks for the feedback!

I will let you know if I find a way to make it more robust.